bot/exts/help_channels/_cog.py

```diff
@@ -387,7 +387,14 @@ async def move_to_dormant(self, channel: discord.TextChannel) -> None:
         )
 
         log.trace(f"Sending dormant message for #{channel} ({channel.id}).")
-        embed = discord.Embed(description=_message.DORMANT_MSG)
+        available_category = self.bot.get_channel(constants.Categories.help_available)
```

Comment:
You should probably check for None and in that case add a log, and default to the old Dormant channels text.

Reply:
I've instead used `utils.channels.try_get_channel`, as suggested by Chrisjl on discord, which will always get the channel